### PR TITLE
Add notes

### PR DESCRIPTION
**Pull request by anandsr21harshit**
- Created Add Notes feature
- Used grid layouts
image

### CODE REVIEW

File: keep-notes/src/pages/Home.jsx [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 function Home() {
+
+  const {state} = useData();
   return (
     <>
       <NavBar/>
-      <main className='home-container'>
+      <header className='home-container'>
         <SideBar/>
         <NotesInput></NotesInput>
```
Comment (rohit8686) [inline keep-notes/src/pages/Home.jsx:14]: you can use self closing tag here

File: keep-notes/src/components/NotesInput.jsx [hunk 2]
```diff
@@ -20,24 +21,28 @@ function NotesInput() {
...
+    try {
+      const response = await axios.post(
+        "/api/notes",
+        { note },
+        { headers: { authorization: token } }
+      );
 
-      if(response.status === 200 || response.status === 201){
-        console.log("check response", response.data.notes)
+      console.log(response);
```
Comment (rohit8686) [inline keep-notes/src/components/NotesInput.jsx:37]: you can remove console statement

**Review by rohit8686 — APPROVED** [review]

**Review by PeeyushPrashant — APPROVED** [review]
looks good

**Review by mdarif6 — APPROVED** [review]
- folder structure short and nice.
- code also clean and easy to read.

**Pull request merged**